libafl/src/generators/mod.rs

```diff
@@ -119,7 +106,7 @@ impl<S> RandBytesGenerator<S> {
 #[derive(Clone, Debug)]
 /// Generates random printable characters
 pub struct RandPrintablesGenerator<S> {
-    max_size: NonZeroUsize,
+    max_size: usize,
```

Comment:
Why would we allow a Generator for 0 bytes?

Reply:
It'll be a bug in 99% of cases?

Reply:
hmmm okay maybe this guy should have NonZero really

Reply:
or i think i should just make this usize const